### ISSUE TITLE
getUSCounty() cannot resolve a lon, lat coordinate

### ISSUE THREAD

**Issue by FHLB** [MazamaScience/MazamaSpatialUtils#22]
I found a longitude, latitude coordinate which yields NA when used with getUSCounty()

R sample code:

## Try out the MazamaSpatialUtils package

library(MazamaSpatialUtils)

setSpatialDataDir("~/Data/Spatial")
## Load US Census County Spatial Data
loadSpatialData("NaturalEarthAdm1")
loadSpatialData("USCensusCounties")

## 42.500429N 71.075913W
lon1 <- -71.075913
lat1 <- 42.500429
state_code1 <- getStateCode(lon1, lat1)
state_code1 ## "MA"
county_name1 <- getUSCounty(lon1, lat1, stateCodes = state_code1)
county_name1 ## "Middlesex"

## getUSCounty() yields NA for this coordinate.
lon2 <- -82.570586
lat2 <- 27.966479
state_code2 <- getStateCode(lon2, lat2)
state_code2 ## "FL"
getUSCounty(lon2, lat2, stateCodes = state_code2) ## NA

**Comment by jonathancallahan** [comment]
Thanks for catching this. This is what happens when searching for a seaside location with a low resolution dataset. Searches are fast but sometimes, come up empty.

The `getStateCode()` and `getCountryCode()` have a `useBuffering` argument that, when TRUE, is used to expand a point location into a small circle to get around this problem. But we forgot to add this argument to `getUSCounty()` (and others?)

We will try to push a solution in the next week.

**Comment by jonathancallahan** [comment]
This has been resolved in version 0.6.3 by adding the `useBuffering` argument to `getUSCounty()`.

Working the above example:

```
> lon2 <- -82.570586
> lat2 <- 27.966479
> state_code2 <- getStateCode(lon2, lat2)
> state_code2 ## "FL"
[1] "FL"
> getUSCounty(lon2, lat2, stateCodes = state_code2, useBuffering = TRUE) ## NA
[1] "Hillsborough"
```
